pricegraph/src/orderbook.rs

```diff
@@ -438,7 +433,8 @@ fn format_path(path: &[NodeIndex]) -> String {
 /// for trades
 fn is_dust_order(element: &Element) -> bool {
     num::is_dust_amount(element.remaining_sell_amount as _)
-        || num::is_dust_amount(num::u256_to_f64(element.balance))
+        || (element.balance < U256::from(u128::MAX)
+            && num::is_dust_amount(element.balance.low_u128()))
```

Comment:
ubernit: Since it is way more likely for balances to be less than `u128::MAX` I would do the fast path first:
```suggestion
        || (num::is_dust_amount(element.balance.low_u128())
             && element.balance < U256::from(u128::MAX))
```

Not sure if this actually has a performance impact though.